src/postorder.rs

```diff
@@ -0,0 +1,51 @@
+//! Fast postorder computation with no allocations (aside from result).
```

Comment:
What license are this file and ssa.rs? Because different files use different licenses I think it would be a good idea to include a license header in each file.

Reply:
The default should be Apache 2 with LLVM exception, but that's a good idea, it would be better to be explicit.